backend/game/cards/discard_cards/health_manipulation_cards/test_heal.py

```diff
@@ -0,0 +1,11 @@
+from game.cards.discard_cards.health_manipulation_cards.heal import Heal
+from game.values import constants
+
+
+def test_heal_adds_2_health(player):
+    Heal().immediate_effect(player, None)
+    assert player.current_health == constants.DEFAULT_HEALTH
```

Comment:
Technically this is correct in that the assert will work if the player is at max/default health it will not add 2 health but this does not test if the player is below the default if it will actually add the two

Reply:
change made and pushed to recent commit

Reply:
Again, technically correct but does suffer some testing flaws.  Should
1. Probably best to use the method in player that adjust the health by a set value
2.  If are going to adjust the health manually need to set it to `player.current_health = constants.DEFAULT_HEALTH - 2` but I recommend 1 personally

Reply:
The above comment has not been addressed